src/helpers/proofs/generateERC721.ts

```diff
@@ -0,0 +1,55 @@
+import { BadgeSourceType } from 'data'
+import { ProofStore } from 'stores/ProofStore'
+import {
+  getEddsaPublicKey,
+  requestAddressOwnershipAttestation,
+  requestBalanceAttestation,
+} from 'helpers/proofs/attestor'
+import { handleError } from '@big-whale-labs/frontend-utils'
+import Network from '@big-whale-labs/stores/dist/models/Network'
+import buildERC721Proof from 'helpers/proofs/buildERC721Proof'
+import checkNavigator from 'helpers/proofs/checkNavigator'
+import getNullifierMessage from 'helpers/proofs/getNullifierMessage'
+import walletStore from 'stores/WalletStore'
+
+export default async function generateERC721(
+  store: ProofStore,
+  original: string,
+  options: {
+    network: Network
```

Comment:
Why don't we put `network` outside of `options`?

Reply:
Because network and secret are not needed everywhere